experimental/devices/mfrc522/blockaccess.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Package mfrc522 controls a Mifare RFID card reader.
+//
+// Datasheet
+//
+// https://www.nxp.com/docs/en/data-sheet/MFRC522.pdf
+package mfrc522
+
+import "fmt"
+
+// BlockAccess defines the block access bits.
+type BlockAccess byte
+
+// SectorTrailerAccess defines the sector trailing block access bits.
+type SectorTrailerAccess byte
+
+// Access bits for the sector data.
+const (
+	AnyKeyRWID    BlockAccess = 0x0  // Any key (A or B) can read, write, increment and decrement block.
+	RAB_WN_IN_DN  BlockAccess = 0x02 // Read (A or B), Write (None), Increment (None), Decrement (None)
+	RAB_WB_IN_DN  BlockAccess = 0x04 // Read (A orB), Write (B), Increment (None), Decrement (None)
+	RAB_WB_IB_DAB BlockAccess = 0x06 // Read (A or B), Write (B), Icrement (B), Decrement (A or B)
+	RAB_WN_IN_DAB BlockAccess = 0x01 // Read (A or B), Write (None), Increment (None), Decrment (A or B)
+	RB_WB_IN_DN   BlockAccess = 0x03 // Read (B), Write (B), Increment (None), Decrement (None)
+	RB_WN_IN_DN   BlockAccess = 0x05 // Read (B), Write (None), Increment (None), Decrement (None)
+	RN_WN_IN_DN   BlockAccess = 0x07 // Read (None), Write (None), Increment (None), Decrement (None)
+)
+
+// Access bits for the sector trail.
+// Every trail sector has the options for controlling the access to the trailing sector bits.
+// For example : KeyA_R[Key]_W[Key]_BITS_R[Key]_W[Key]_KeyB_R[Key]_W[Key]
+//
+// - KeyA
+//   - could be Read by providing [Key] ( where [Key] could be KeyA or KeyB )
+//   - could be Written by Providing [Key] ( where [Key] is KeyA or KeyB )
+// - access bits for the sector data (see above)
+//   - could be Read by providing [Key] ( where [Key] could be KeyA or KeyB )
+//   - could be Written by Providing [Key] ( where [Key] is KeyA or KeyB )
+// - KeyB
+//   - could be Read by providing [Key] ( where [Key] could be KeyA or KeyB )
+//   - could be Written by Providing [Key] ( where [Key] is KeyA or KeyB )
+//
+// example:
+//
+//  KeyA_RN_WA_BITS_RA_WA_KeyB_RA_WA means
+//  - KeyA could not be read but could be overwriten if KeyA is provided
+//  - Access bits could be read and overwritten if KeyA is provided during the card authentication
+//  - KeyB could be read and overriten if KeyA is provided during the card authentication
+// more on the matter: https://www.nxp.com/docs/en/data-sheet/MF1S50YYX_V1.pdf
+const (
+	KeyA_RN_WA_BITS_RA_WN_KeyB_RA_WA        SectorTrailerAccess = 0x0
```

Comment:
Same here, this is exposing internal bits while in practice you could write it in a more meaningful way by not using the acronyms. This can be done later.